frontend/src/components/dashboard/Dashboard.tsx

```diff
@@ -5,14 +5,16 @@ import ModalContextProvider from 'contexts/modal.context'
 import CampaignContextProvider from 'contexts/campaign.context'
 import FinishLaterContextProvider from 'contexts/finish-later.modal.context'
 import Error from 'components/error'
-import { NavBar } from 'components/common'
+import { InfoBanner, NavBar } from 'components/common'
 import Campaigns from './campaigns'
 import Create from './create'
 import Settings from './settings'
+import { INFO_BANNER } from 'config'
 
 const Dashboard = () => {
   return (
     <ModalContextProvider>
+      <InfoBanner>{INFO_BANNER}</InfoBanner>
```

Comment:
Should the InfoBanner be outside the `ModalContextProvider`? So that the banner is still visible when the modals are shown.

Reply:
I think it should be ok to not show the banner with the modal. Including it adds scroll to the modal background.

Reply:
Alright!